MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -52153,200 +52363,340 @@
     },
     {
       "Id": "CollectableDekuPlaygroundItem1",
-      "RequiredItems": [
-        "OtherPlayDekuPlayground"
+      "RequiredItems": [],
+      "ConditionalItems": [
+        [
+          "OtherPlayDekuPlayground"
+        ],
+        [
+          "Deku Playground Rupee Displacement",
+          "Ocarina of Time or a Transformation Mask"
```

Comment:
How come you didn't make `Ocarina of Time or a Transformation Mask` a requirement for `Deku Playground Rupee Displacement` or just put those items directly into the trick? Also I think you can displace the rupees using any cutscene item, e.g. Magic Beans. Am I wrong or was that purposely omitted?

Reply:
First question: Tricks cannot be conditionals for other tricks, and Fierce Deity's Mask Anywhere is the trick preventing me from simply adding all of them as conditionals for it. Second question: I was not aware of that! I will gladly rewrite this in a future logic revision.

Reply:
Actually, it seems you can have tricks as conditionals for the entries that tricks require? Maybe this can be cleaned up.

Reply:
Cleaned it up and added trade items for Rupee Displacement in revision 2. Magic Beans cannot be used at all in Deku Playground.